### PR TITLE
Activate preset async

### LINKED ISSUE #346: performance not recalculating after resources (CPU, RAM) reallocation
change to CPU cores allocation impacts node's performance, but now change CPU cores or RAM will not trigger performance recalculation. This leads to wrong perception of node as a provider.

needs to be fixed before golemfactory/golem#2771

### PR DESCRIPTION
**Pull request by tworec**
`activate_hw_preset` is changed to be async, and return performance values
fixes golemfactory/golem-electron#346
works only with golemfactory/golem-electron#347

### CODE REVIEW

**Review by prekucki — APPROVED** [review]

File: golem/client.py [hunk 1]
```diff
@@ -1126,7 +1126,7 @@ def get_environments(self):
 
     @inlineCallbacks
     def run_benchmark(self, env_id):
-        logger.info('Running benchmarks ...')
+        logger.info('Running benchmark for %s', env_id)
```
Comment (maaktweluit) [inline golem/client.py]: This log line is now duplicated ( benchmarkmanager.py ), maybe "requesting benchmark" followed by "running benchmark"?

**Review by maaktweluit — COMMENTED** [review]
LGTM! Small comment, will approve on reply

File: golem/client.py (hunk 1 above)
Comment (tworec) [reply golem/client.py]: ok, I'll remove one of them

**Comment by tworec** [comment]
related to #2771

**Pull request merged**